modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js

```diff
@@ -270,6 +270,14 @@ export default class Blogs {
 		return document.getElementById(`${this._config.namespace}${id}`);
 	}
 
+	_getValuesByName(name) {
+		const nodes = document.querySelectorAll(
+			`input[name=${this._config.namespace + name}]`
+		);
+
+		return [...nodes].map((node) => node.value);
+	}
```

Comment:
I think that we don't need a collection here, if we have inputs with the same name we have another issue.
This function can be updated as:

```js
	_getValuesByName(name) {
		return document.querySelector(`input[name=${this._config.namespace}${name}]`)?.value || '';
	}
```